docs/source/how_to_guides/configure_cloud_storage_credentials.md

```diff
@@ -251,6 +252,19 @@ export AZURE_ACCOUNT_ACCESS_KEY='NN1KHxKKkj20ZO92EMiDQjx3wp2kZG4UUvfAGlgGWRn6sPR
 ```
 ````
 
+## Huggingface Datasets
+
+To authenticate with uploading and for private downloads, set the `HF_TOKEN` beforehand.
```

Comment:
Instead of 
"To authenticate with uploading and for private downloads, set the `HF_TOKEN` beforehand.", 

how about 

"To authenticate Huggingface Hub access, users must set their HuggingFace token ([[HF_TOKEN](https://huggingface.co/docs/huggingface_hub/main/en/package_reference/environment_variables#hftoken)](https://huggingface.co/docs/huggingface_hub/main/en/package_reference/environment_variables#hftoken)) in the run environment. See the [[HF's documentation](https://huggingface.co/docs/huggingface_hub/guides/hf_file_system)](https://huggingface.co/docs/huggingface_hub/guides/hf_file_system) on the URL format."

Reply:
Thanks for the documentation help! Added.